Makefile

```diff
@@ -319,6 +319,10 @@ clean:
 	@$(MAKE) -sC $(srcdir)/doc clean
 	@$(MAKE) -sC $(srcdir)/libtraceevent clean
 
+gcov-clean:
+	$(Q)find -name "*\.gcda" | xargs $(RM)
+	$(Q)$(RM) coverage.info
+
```

Comment:
Then why not just `make clean` having dependency of `make gcov-clean` in the previous commit?

Reply:
It's because `make gcov-clean` removes only `.gcda` files so that another execution can generate coverage info normally. If `.gcno` files are also removed, we have to recompile uftrace.
And I may have to change this rule to `make coverage-clean` if you think it's better.